src/code/reducers/index.js

```diff
@@ -21,7 +21,7 @@ function initialState() {
             visibleGenes: [],
             hiddenAlleles: [],
             trial: 0,
-            routeSpec: {level: 0, mission: 0, challenge: 0},
+            routeSpec: null,
```

Comment:
This was another significant change - having the `routeSpec` initialize to `{0, 0, 0}` was causing loading errors. Basically, if you went to the actual level `{0, 0, 0}`, and reloaded the page, nothing would render. This was because (through a recent change we made), if the `routeSpec` hasn't changed, we don't load a level. So, now I initialize the `routeSpec` to null instead.